### PR TITLE
Cancel floating on top during video pausing.

### PR DESCRIPTION
**Pull request by neesonqk**
Implementation #133

### CODE REVIEW

**Comment by lhc70000** [comment]
Please don't do check in `keyDown`.

For querying status please use `playerCore.info` or `MPVController#propertyChanged` or getting mpv property at runtime, since user may pause by menu actions, lua scripts, JSON IPCs, etc.

Since it's not a normal behavior, I think a better implementation should:

- Connect `--ontop` with window floating status (like what `fullscreen` does now)
- Add a user option about whether floating on top when paused
- Set `ontop` to false when paused and above option enabled

Please continue discussion in the issue if needed.

**Before working on a PR, please read [CONTRIBUTING.md](https://github.com/lhc70000/iina/blob/master/CONTRIBUTING.md).**
TL;DR:
- Open an issue if there isn't one
- Ask for the author's permission in the issue

**Comment by neesonqk** [comment]
Seems I didn't violate the rules, `fullscreen` does the similar way I suppose.

There are two implementations of pausing a video, one is through clicking pause icon on the control bar, another is through SPACE key, they seem to use different mechanisms?

By clicking pause from control bar can be implemented almost exactly the same as fullscreen, but by tapping SPACE key, if we don't intercept the `SPACE` key in `keyDown` event, I cannot image a better way to temporarily toggle off/on floating top while pausing was enabled by SPACE key.

**Comment by lhc70000** [comment]
> There are two implementations of pausing a video

No, there are many other ways to change a mpv property (including pause) like other key bindings, lua script, JSON IPC or set by mpv itself, and we cannot detect these changes since they are not triggered by IINA.

So we must listen to certain property change event and synchronize with window status, take full screen as example: [c2ceedf](https://github.com/lhc70000/iina/commit/c2ceedf6118d9593dd29878130e5552f8c9bd320).

**Comment by neesonqk** [comment]
Right I got your purpose.
